### DISCUSSION TITLE
Order of the CVD fields

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#896] [Ideas]
In my opinion, the order of the CVD fields could be improved. Due to the conditions, the category must be specified first, then the type and finally the number of vehicles. However, the order of the fields is currently completely reversed:

Current notice-type definition:
{
            "id" : "GR-LotResult-CVD-Vehicles",
            "contentType" : "group",
            "nodeId" : "ND-ProcurementDetailsLotResult",
            "displayType" : "GROUP",
            "description" : "Information about the assets: category and quantity (number) in the context of an applicable Legal Basis",
            "_label" : "group|name|ND-ProcurementDetailsLotResult",
            "_repeatable" : true,
            "content" : [ {
              "id" : "GR-LotResult-CVD-Vehicles-Statistics",
              "contentType" : "group",
              "nodeId" : "ND-ProcurementStatistics",
              "displayType" : "GROUP",
              "description" : "Numbers about the procurement: category (code) and quantity (number)",
              "_label" : "group|name|ND-ProcurementStatistics",
              "_repeatable" : true,
              "content" : [ {
                "id" : "OPT-156-LotResult",
                "contentType" : "field",
                "displayType" : "TEXTBOX",
                "description" : "Vehicle Numeric",
                "_label" : "field|name|OPT-156-LotResult"
              }, {
                "id" : "OPT-155-LotResult",
                "contentType" : "field",
                "displayType" : "COMBOBOX",
                "description" : "Vehicle Type",
                "_label" : "field|name|OPT-155-LotResult"
              } ]
            }, {
              "id" : "BT-723-LotResult",
              "contentType" : "field",
              "displayType" : "COMBOBOX",
              "description" : "Vehicle Category",
              "_label" : "field|name|BT-723-LotResult"
            } ]
          }

In my opinion, the following order would be much better. Both in terms of entering the values and logically:
{
            "id" : "GR-LotResult-CVD-Vehicles",
            "contentType" : "group",
            "nodeId" : "ND-ProcurementDetailsLotResult",
            "displayType" : "GROUP",
            "description" : "Information about the assets: category and quantity (number) in the context of an applicable Legal Basis",
            "_label" : "group|name|ND-ProcurementDetailsLotResult",
            "_repeatable" : true,
            "content" : [  {
              "id" : "BT-723-LotResult",
              "contentType" : "field",
              "displayType" : "COMBOBOX",
              "description" : "Vehicle Category",
              "_label" : "field|name|BT-723-LotResult"
            }, {
              "id" : "GR-LotResult-CVD-Vehicles-Statistics",
              "contentType" : "group",
              "nodeId" : "ND-ProcurementStatistics",
              "displayType" : "GROUP",
              "description" : "Numbers about the procurement: category (code) and quantity (number)",
              "_label" : "group|name|ND-ProcurementStatistics",
              "_repeatable" : true,
              "content" : [ {
                "id" : "OPT-155-LotResult",
                "contentType" : "field",
                "displayType" : "COMBOBOX",
                "description" : "Vehicle Type",
                "_label" : "field|name|OPT-155-LotResult"
              }, {
                "id" : "OPT-156-LotResult",
                "contentType" : "field",
                "displayType" : "TEXTBOX",
                "description" : "Vehicle Numeric",
                "_label" : "field|name|OPT-156-LotResult"
              } ]
            } ]
          }

**Comment by YvesJo** [comment]
Hi,
Thanks for sharing this suggestion.
Fields reorganization has been addressed and will be part of SDK 1.12 release.
KR

**Reply by JanEmde** [reply]
Thanks!

Another group worth improving is GR-LotTender-Subcontracting. BT-773-Tender should come first here.

**Reply by YvesJo** [reply]
Thanks for reporting, this will be updated in SDK 1.12
